cube_holder.scad

```diff
@@ -17,29 +17,39 @@ cube_holder();
 // Begin modules that can be used to assemble this piece in solver_3.scad
 
 module above_cube_holder() {
-  up(thickness/2 + servo_gear_thickness) above_frame() children();
+  up(servo_gear_thickness) above_frame() children();
 }
 
 module cube_holder() {
+  up(servo_gear_thickness) onto_point()
```

Comment:
this should be moved into solver_3.scad, so that line 15's `cube_holder()` is properly oriented for printing